### PR TITLE
[CMS-711] WordPress 6.0 release notes

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

Adds release notes for WordPress 6.0 release, and removes 6.0 blurb from known issues page.

### Dependencies and Timing

- [ ] Site Repository Tool [PR #9](https://github.com/pantheon-systems/site-repository-tool/pull/9)

**Release**:
- [ ] After dependencies above are complete

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7385](https://pr-7385--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `56.99999999999999` | `7` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7385` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37902/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37902/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7385--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by jspellman814** [comment]
[Preview](https://pr-7385--pantheon-docs.my.pantheonfrontend.website/docs/start-states/wordpress#20220524-1)

**Review by namespacebrian — APPROVED** [review]
I approve this message

File: source/content/start-states/wordpress.md [hunk 1]
```diff
@@ -7,13 +7,23 @@ tags: [code, site, upstreams]
...
 
 For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-systems/WordPress) follows [WordPress core](https://wordpress.org/news/category/releases/) one-to-one. This document is intended to provide further context to platform-specific changes in Pantheon's WordPress upstream.
 
 ## Latest Release
 
+### 2022-05-24
+
+#### <a name="20220524-1" class="release-update"></a>WordPress 6.0
+
+If you’ve updated the [Twenty Twenty-Two theme](https://wordpress.org/themes/twentytwentytwo/) outside of Pantheon's WordPress upstream, there are conflicting updates from the WordPress core repository.
```
Comment (greg-1-anderson) [inline source/content/start-states/wordpress.md:21]: Maybe:
> If you have updated the [Twenty Twenty-Two theme](https://wordpress.org/themes/twentytwentytwo/) theme bundled with Pantheon's WordPress upstream, for example with Autopilot or WP-CLI, then your site has conflicting commits with the latest WordPress 6.0 release.

File: source/content/start-states/wordpress.md [hunk 2]
```diff
@@ -7,13 +7,23 @@ tags: [code, site, upstreams]
...
+
+Pantheon will automatically resolve this conflict when the `Apply Updates` button is clicked on your dashboard by [removing](https://core.trac.wordpress.org/changeset/53286) `wp-content/themes/twentytwentytwo/assets/fonts/LICENSE.md`
```
Comment (greg-1-anderson) [inline source/content/start-states/wordpress.md:23]: "these conflicts"

**Comment by greg-1-anderson** [comment]
LGTM, ready for docs team review. Prereq PR should go to production tomorrow morning.

**Pull request merged**